Example/OriginateUI/AppDelegate.m

```diff
@@ -7,21 +7,33 @@
 //
 
 #import "AppDelegate.h"
-#import "MyViewController.h"
+#import "HomeViewController.h"
 
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
     self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
-
-    MyViewController *themableViewController = [[MyViewController alloc] init];
-    UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:themableViewController];
-    
+    HomeViewController *homeViewController = [[HomeViewController alloc] init];
+    UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:homeViewController];
     self.window.rootViewController = navigationController;
-    
+
+    navigationController.navigationBar.topItem.title = @"OriginateUI";
```

Comment:
In order to set the title just set the title property of your home view controller. In addition when using localized strings always use NSLocalizedKey to allow easy change for future localization.